Roslyn.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <RunConfiguration>  
+    <TestAdaptersPaths>%userprofile%\.nuget\packages\xunit.runner.visualstudio\2.2.0-beta4-build1194</TestAdaptersPaths>  
```

Comment:
> If not, perhaps I could add a comment that this file can be removed once #20333 is merged.

Lets do that.  I can just sync this change in my branch later tonight and delete the file immediately as a part of #20333